### ISSUE TITLE
Visualize flag in slurm scripts does not exist

### ISSUE THREAD

**Issue by tomsynativ** [vishalned/MMEarth-train#3]
When running `./slurm_scripts/slurm_fine_tune_seg.sh`, I ran into the following error:

`usage: FCMAE fine-tuning [-h] [--batch_size BATCH_SIZE] [--epochs EPOCHS] [--update_freq UPDATE_FREQ] [--model MODEL] [--input_size INPUT_SIZE] [--drop_path PCT] [--layer_decay_type {single,group}]
                         [--patch_size PATCH_SIZE] [--baseline BASELINE] [--model_ema MODEL_EMA] [--model_ema_decay MODEL_EMA_DECAY] [--model_ema_force_cpu MODEL_EMA_FORCE_CPU]
                         [--model_ema_eval MODEL_EMA_EVAL] [--clip_grad NORM] [--weight_decay WEIGHT_DECAY] [--lr LR] [--blr LR] [--layer_decay LAYER_DECAY] [--min_lr LR] [--warmup_epochs N] [--warmup_steps N]
                         [--opt OPTIMIZER] [--opt_eps EPSILON] [--opt_betas BETA [BETA ...]] [--momentum M] [--weight_decay_end WEIGHT_DECAY_END] [--color_jitter PCT] [--aa NAME] [--smoothing SMOOTHING]
                         [--train_interpolation TRAIN_INTERPOLATION] [--reprob PCT] [--remode REMODE] [--recount RECOUNT] [--resplit RESPLIT] [--mixup MIXUP] [--cutmix CUTMIX]
                         [--cutmix_minmax CUTMIX_MINMAX [CUTMIX_MINMAX ...]] [--mixup_prob MIXUP_PROB] [--mixup_switch_prob MIXUP_SWITCH_PROB] [--mixup_mode MIXUP_MODE] [--finetune FINETUNE]
                         [--head_init_scale HEAD_INIT_SCALE] [--model_key MODEL_KEY] [--model_prefix MODEL_PREFIX] [--linear_probe LINEAR_PROBE] [--data_path DATA_PATH] [--nb_classes NB_CLASSES]
                         [--output_dir OUTPUT_DIR] [--log_dir LOG_DIR] [--device DEVICE] [--seed SEED] [--resume RESUME] [--eval_data_path EVAL_DATA_PATH]
                         [--imagenet_default_mean_and_std IMAGENET_DEFAULT_MEAN_AND_STD]
                         [--data_set {CIFAR,IMNET,image_folder,geobench.m-eurosat,geobench.m-so2sat,geobench.m-bigearthnet,geobench.m-brick-kiln,geobench.m-cashew-plantation,geobench.m-SA-crop-type}]
                         [--auto_resume AUTO_RESUME] [--save_ckpt SAVE_CKPT] [--save_ckpt_freq SAVE_CKPT_FREQ] [--save_ckpt_num SAVE_CKPT_NUM] [--start_epoch N] [--eval EVAL] [--dist_eval DIST_EVAL]
                         [--num_workers NUM_WORKERS] [--pin_mem PIN_MEM] [--crop_pct CROP_PCT] [--world_size WORLD_SIZE] [--local_rank LOCAL_RANK] [--dist_on_itp DIST_ON_ITP] [--dist_url DIST_URL]
                         [--use_amp USE_AMP] [--wandb WANDB] [--wandb_project WANDB_PROJECT] [--wandb_run_name WANDB_RUN_NAME] [--pretraining PRETRAINING] [--use_orig_stem USE_ORIG_STEM]
                         [--run_on_test RUN_ON_TEST] [--percent PERCENT] [--num_samples NUM_SAMPLES] [--test_scores_dir TEST_SCORES_DIR]
FCMAE fine-tuning: error: unrecognized arguments: --visualize False`

I have removed the flag from the script and that seemed to work.

**Comment by vishalned** [comment]
should be fixed now
